config/templates/cloud-config-worker

```diff
@@ -233,9 +197,6 @@ coreos:
         [Service]
         Type=oneshot
         ExecStartPre=/usr/bin/bash -c "while sleep 1; do if /usr/bin/curl  --insecure -s -m 20 -f  https://127.0.0.1:10250/healthz > /dev/null ; then break ; fi;  done"
-        {{ if .UseCalico }}
-        ExecStartPre=/usr/bin/systemctl is-active calico-node
-        {{ end }}
```

Comment:
Any idea how we could hold on/delay running cfn-sginal until self-hosted calico becomes ready?